test/test_other.py

```diff
@@ -12500,3 +12498,28 @@ def test_fs_icase(self):
 
   def test_strict_js_closure(self):
     self.do_runf(test_file('hello_world.c'), emcc_args=['-sSTRICT_JS', '-Werror=closure', '--closure=1', '-O3'])
+
+  def test_em_js_deps(self):
+    # Check that EM_JS_DEPS works. Specifically, multiple different instances in different
+    # object files.
+    create_file('f1.c', '''
+    #include <emscripten.h>
+
+    EM_JS_DEPS(other, "$allocateUTF8OnStack");
```

Comment:
Where's `other` function?

Reply:
The first argument here is just a unique identifier for the dependencies.  I forgot to document that fully.